### ISSUE TITLE
[7.0.0]vCan't import flow types

### ISSUE THREAD

**Issue by stunaz** [erikras/redux-form#3175]
Hi, I am trying to import the new outstanding types in my project, but it doesnt seem to work .
I am trying this way : 
`import type { FormProps } from 'redux-form';`

Unless there is another way to import them.
I also noticed a bunch of errors in my editors (atom & vscode):

```
file: 'file:../node_modules/redux-form/lib/index.js'
severity: 'Error'
message: 'inconsistent use of library definitions (Object.defineProperty Property not found in object literal)'
at: '1,1'
source: 'flow'
```

<img width="957" alt="screen shot 2017-07-13 at 10 20 17 am" src="https://user-images.githubusercontent.com/1305483/28171157-8db06eca-67b5-11e7-9819-59d5a59875c7.png">

**Comment by erikras** [comment]
Ooh, I _did_ want to export that, but I don't think I got around to it. What I am assuming we are talking about is the type that would match [this object](http://redux-form.com/7.0.0/docs/api/Props.md/), the props provided to your form component from the redux form HOC..

**Comment by stunaz** [comment]
yes , I intend to use it as the following:

```javascript
// @flow

import React, { Component } from 'react';
import {Field, reduxForm } from 'redux-form';

import type { FormProps } from 'redux-form';

type Props = FormProps & {
  onSubmitProp: Function,

};

class ContactForm extends Component {
  props: Props;

  render() {
    const { handleSubmit, pristine, submitting, onSubmitProp } = this.props;

    return (
      <form onSubmit={handleSubmit(this.onSubmitProp)}>

          <Field
              name="name"
              component="input"
              placeholder="Name"
            />

          <button type="submit" disabled={pristine || submitting}>
      </form>
    );
  }

}

export default reduxForm({
  form: 'contact'
})(ContactForm)
```

**Comment by erikras** [comment]
Fix published in [`v7.0.1`](https://github.com/erikras/redux-form/releases/tag/v7.0.1).
